pom.xml

```diff
@@ -60,6 +60,10 @@
       <artifactId>vertx-docgen</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
this project cannot depend on vertx-web

Reply:
Oh I see, thank @vietj I will open the PR in vertx-web-client